openSpotServer/parkingSpot/parkingSpots.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+
+# 					  [x,y]
+#                     Top Left, Top Right,  Bottom Right, Bottom Left
+spot_15 = np.array([[1465,152], [1700,152], [1700,248],   [1430,248]], np.int32)
```

Comment:
How about storing this as an array of spots so we dont have to manually iterate through each individual spot  like we do in `openSpotServer/parkingSpot/DEMO_ParkingSpotMask.py` and we can just iterate through the array of spots and perform what ever operation since we will be performing the same operation on each one.

Reply:
This is just for testing purpose. We will do array after the mount gets 3D printed. Hows the progress on 3D printing?